pepys_timeline/static/js/src/pepys.js

```diff
@@ -85,8 +88,13 @@ function fetchSerialsMeta() {
 
     const url = new URL(window.location + 'dashboard_metadata');
     const queryParams = new URLSearchParams();
-    queryParams.set('from_date', '2021-01-05');
-    queryParams.set('to_date', '2021-01-05');
+
+    const today = new Date();
+    const yesterday = new Date();
+    yesterday.setDate(today.getDate() - 1);
+
+    queryParams.set('from_date', moment(yesterday).format(DATE_FORMATS.metadata));
+    queryParams.set('to_date', moment(yesterday).format(DATE_FORMATS.metadata));
```

Comment:
the above may be quite correct, but...

Logically, I'd expect yesterdays coverage to go from 00:00 from_date until 00:00 to_date - so line 97 would use `today`.

But, I recall a conversation with @arunlal-v  where his SQL logic interprets the start & end dates differently - from 00:00 on `from` until 23:59 on `to`. 

Aah, yes.  The deleted code above shows this this special processing. 

I'll capture this modification in a separate issue, and accept this PR :-)